docs/howto/specify-splits.md

```diff
@@ -0,0 +1,48 @@
+# Howto specify ML data splits
+
+ML datasets may come in [different data splits, intended to be used for different steps of a model building](https://en.wikipedia.org/wiki/Training,_validation,_and_test_data_sets), usually training, validation and testing.
+
+The Croissant format allows for the data to be split arbitrarily into one or multiple splits, which for example allows dataset consumers to load a specific split.
+
+## The split field
+
+The splits definition is done at the `RecordSet` level, with a dedicated field which can be named "split" or anything else, but must have the semantic data type [`wd:Q3985153`](https://www.wikidata.org/wiki/Q3985153). That field data type is generally `sc:Text`, but could theoritically come as other data types, `sc:Integer` for example.
+
+The following artificial example defines the split(s) based on a CSV column:
+
+TODO: example snippet, also with `wd:` context declaration.
+
+Often, data from different splits come packaged into different files, which enable dataset consumers to only download the needed data. In such cases, the split definition is still done at the `RecordSet` level, but can refer to data file names or path.
+
+The following example is extracted from the [COCO2014 dataset Croissant config](https://github.com/mlcommons/datasets_format/blob/main/datasets/coco2014/metadata.json):
+
+<https://github.com/mlcommons/datasets_format/blob/2e05069f02a93e6154137aa267e69bd1271a95b0/datasets/coco2014/metadata.json#L173-L187>
```

Comment:
Code snippets should render like https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-a-permanent-link-to-a-code-snippet, but I don't know how to preview that...